rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -31,14 +31,29 @@
     from rotkehlchen.accounting.mixins.event import AccountingEventMixin
     from rotkehlchen.accounting.pot import AccountingPot
     from rotkehlchen.fval import FVal
+    from rotkehlchen.types import Fee
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
-class AssetMovement(HistoryBaseEntry):
+class AssetMovementExtraData(TypedDict):
+    """Typed dict with all the valid fields used in extra_data for AssetMovements"""
+    # Address receiving or sending funds in the asset movement.
+    address: NotRequired[str]
+    # Transaction if it is available where funds were moved on chain.
+    transaction_id: NotRequired[str]
+    # Internal reference used in exchanges. Can be either an id or a url.
```

Comment:
what do you mean by either an id or a url? This needs to be the unique exchange id reference. Not a link

Reply:
Fixed it in coinbase where we were using a url here